source/keymap.js

```diff
@@ -12,7 +12,8 @@
 			19  : 'pause',
 			412 : 'rewind',
 			417 : 'fastforward',
-			461 : 'back'
+			461 : 'back',
+			8	: 'back'	//webkit keycode of backspace key in keyboard
```

Comment:
Hi @DavidUM yep no disagreement there - I think my point was that this is not a webOS-specific keyCode.